src/ocm/components/clusterDetail/ClusterDetail.tsx

```diff
@@ -84,6 +85,23 @@ const ClusterDetail: React.FC<ClusterDetailProps> = ({ cluster }) => {
               }
             />
           </RenderIf>
+          <RenderIf condition={cluster.status == 'installed'}>
```

Comment:
```suggestion
          <RenderIf condition={cluster.status === 'installed'}>
```

Reply:
Also as per the bug description, "this message shouldn't show when the installed cluster is an SNO cluster"